### ISSUE TITLE
t/10-hpc.t NBI::Slurm::queues()

### ISSUE THREAD

**Issue by dmr3** [quadram-institute-bioscience/NBI-Slurm#1]
This is a very exciting module.  Thank you!

installing via cpanminus i ran into a test fail

```
Undefined subroutine &NBI::Slurm::queue called at t/10-hpc.t line 13.
t/10-hpc.t ........................
Dubious, test returned 255 (wstat 65280, 0xff00)
```

it just needed an update on the method called to grab the queues. 

```
my @queue = NBI::Slurm::queues();
ok(scalar @queue > 0, "queues() returns a list of queues");
```

**Comment by telatin** [comment]
Ups, I have a set of local tests where the HPC is available, I need to check what happens there :)

UPDATE: Typo fixed, thanks you a million for reporting!

**Comment by telatin** [comment]
Hi @dmr3 , thanks for reporting, the fix is available on as of  [v0.8.2](https://metacpan.org/release/PROCH/NBI-Slurm-0.8.2)
As you can see this module is mostly to automate tasks in the cluster I use, but I released as some commands like  `runjob` or `waitjobs` might be handy elsewhere.
Should you have suggestions, feel free to let me know :)
